h2o-algos/src/main/java/hex/gam/GAM.java

```diff
@@ -465,6 +465,8 @@ private void copyGLMtoGAMModel(GAMModel model, GLMModel glmModel) {
         model._output._glm_validation_metrics = glmModel._output._validation_metrics;
       model._output._glm_scoring_history = model.copyTwoDimTable(glmModel._output._scoring_history);
       model._output._glm_model_summary = model.copyTwoDimTable(glmModel._output._model_summary);
+      model._output._glm_scoring_history_early_stop = glmModel._output._scoring_history_early_stop==null?null:
+              model.copyTwoDimTable(glmModel._output._scoring_history_early_stop);
       if (_parms._family == multinomial || _parms._family == ordinal) {
```

Comment:
I need access to scoring_history.  However, GLM already has its own, so I created one just for early_stop

Reply:
I dont undestant the comment, scoring history is there for the reason of early stopping, so why do you need another one? how are they different?

Reply:
@honzasterba The GLM scoring history is totally different from the scoring history that is used for early stopping.  Since it has been there for a long time, I don't want to just remove it.  I added a new one for early stopping.  There are many differences between GLM and the other Algos.  It is before my time at H2O.

Reply:
whats the original scoring history used for? can we somehow unify? it seems wasteful to have two, I thought that scoring is done exactly for the reason of early stopping

Reply:
Nope.  GLM is written very differently from the others.  The glm scoring history is very light weight and very simple.  Since it is there for a very long time, I don't want to change it to break some customer code.

Reply:
I have combined the two scoring_history into one with the original GLM scoring history coming first.  Hope this minimizes breaking anyone's code